share/dss/dss_test.go

```diff
@@ -0,0 +1,224 @@
+package dss
+
+import (
+	"crypto/rand"
+	"testing"
+
+	"github.com/dedis/crypto/abstract"
+	"github.com/dedis/crypto/ed25519"
+	"github.com/dedis/crypto/eddsa"
+	"github.com/dedis/crypto/random"
+	"github.com/dedis/crypto/share/dkg"
+	"github.com/dedis/crypto/share/vss"
+	"github.com/dedis/crypto/sign"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+var suite = ed25519.NewAES128SHA256Ed25519(false)
+
+var nbParticipants = 7
+var t = nbParticipants/2 + 1
+
+var partPubs []abstract.Point
+var partSec []abstract.Scalar
+
+var longterms []*dkg.DistKeyShare
+var randoms []*dkg.DistKeyShare
+
+var dss []*DSS
+
+func genPair() (abstract.Scalar, abstract.Point) {
+	sc := suite.Scalar().Pick(random.Stream)
+	return sc, suite.Point().Mul(nil, sc)
+}
+
+func randomBytes(n int) []byte {
+	var buff = make([]byte, n)
+	rand.Read(buff[:])
+	return buff
+}
+
+func init() {
```

Comment:
style: please put `init` on top, just after `suite` and `nbParticipants`. Then the `Test*` and the private helpers at the bottom.